From: Stephen Kelly <steveire@gmail.com>
Date: Mon, 20 Jul 2015 18:03:47 +0000
Subject: Remove policy settings from FindKDE4Internal.
X-Git-Tag: v4.14.11
X-Git-Url: http://quickgit.kde.org/?p=kdelibs.git&a=commitdiff&h=ddd2b3290d5d7cef9abfba7ce5e15b6c801d531c
---
Remove policy settings from FindKDE4Internal.

At this point, the ones which are set here are all set to NEW, except
CMP0011.  The point of CMP0011 here is to make the policy settings
be used by consumers.  All consumers need to gain a use of
the cmake_minimum_required command now anyway to satisfy CMP0000, so
just remove the call in the internal file.
---


--- a/cmake/modules/FindKDE4Internal.cmake
+++ b/cmake/modules/FindKDE4Internal.cmake
@@ -345,35 +345,6 @@
 # Redistribution and use is allowed according to the terms of the BSD license.
 # For details see the accompanying COPYING-CMAKE-SCRIPTS file.
 
-
-# this is required now by cmake 2.6 and so must not be skipped by if(KDE4_FOUND) below
-cmake_minimum_required(VERSION 2.8.9 FATAL_ERROR)
-# set the cmake policies to the 2.4.x compatibility settings (may change for KDE 4.3)
-cmake_policy(VERSION 2.4.5)
-
-# CMake 2.6, set compatibility behaviour to cmake 2.4
-# this must be executed always, because the CMAKE_MINIMUM_REQUIRED() command above
-# resets the policy settings, so we get a lot of warnings
-
-# CMP0003: add the link paths to the link command as with cmake 2.4
-cmake_policy(SET CMP0003 NEW)
-
-cmake_policy(SET CMP0005 NEW)
-# since cmake 2.6.3: NEW behaviour is that setting policies doesn't "escape" the file
-# where this is done, macros and functions are executed with the policies as they
-# were when the were defined. Keep the OLD behaviour so we can set the policies here
-# for all KDE software without the big warning
-cmake_policy(SET CMP0011 OLD)
-
-# since cmake 2.8.4: when include()ing from inside cmake's module dir, prefer the files
-# in this directory over those from CMAKE_MODULE_PATH
-cmake_policy(SET CMP0017 NEW)
-
-if (POLICY CMP0026)
-  # Don't use the LOCATION target property of buildsystem targets.
-  cmake_policy(SET CMP0026 NEW)
-endif (POLICY CMP0026)
-
 # Only do something if it hasn't been found yet
 if(NOT KDE4_FOUND)
 

